webapp/src/components/SingleStudy/NoteView/index.tsx

```diff
@@ -104,16 +106,18 @@ const NoteView = (props: Props) => {
   );
   const [editionMode, setEditionMode] = useState<boolean>(false);
   const [content, setContent] = useState<string>('');
+  const [loaded, setLoaded] = useState(false);
 
   const onSave = async (newContent: string) => {
     try {
       await editComments(studyId, newContent);
       setEditorState(EditorState.createWithContent(convertXMLToDraftJS(newContent)));
       setContent(newContent);
-      setEditionMode(false);
       enqueueSnackbar(t('singlestudy:commentsSaved'), { variant: 'success' });
     } catch (e) {
       enqueueErrorSnackbar(enqueueSnackbar, t('singlestudy:commentsNotSaved'), e as AxiosError);
+    } finally {
+      setEditionMode(false);
```

Comment:
keep this in the try block, as we want to exit the edition mode only if saving was ok (to keep current changes)
if it keeps failing, the user can quit the edition mode with another button.